.github/workflows/release.yml

```
@@ -156,6 +156,29 @@ jobs:
             phylum-*.zip
             phylum-*.zip.minisig
 
+  Trigger:
+    name: Trigger phylum-ci Docker image creation
+    needs: Release
+    # The `--fail-with-body` option in `curl` was added in v7.76.0, which is too
+    # new for the `ubuntu-latest` runner that currently makes use of Ubuntu 20.04.
+    # TODO: Update to `ubuntu-latest` once `ubuntu-22.04` support is stabilized.
+    #       https://github.com/phylum-dev/cli/issues/467
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Trigger phylum-ci Docker image creation
+        # Don't trigger for pre-releases
+        if: contains(github.ref, 'rc') != true
```

As mentioned in phylum-dev/phylum-ci#65, I think this option is more readable:

```suggestion
        if: ${{ !contains(github.ref, 'rc') }}
```

---

`!= true` might look kinda silly, but if it works, it works. Only suggestion I could make is that `== false` looks less like a bug.